product/src/main/resources/camel-quarkus-product-source.json

```diff
@@ -620,6 +620,7 @@
             "includes": [
                 "integration-tests/*/pom.xml",
                 "integration-test-groups/*/*/pom.xml",
+                "integration-test-groups/*/*/*/pom.xml",
```

Comment:
@ppalaga I just wonder if we can use `integration-test-groups/**/pom.xml` to includes all of the group tests?

Reply:
The `**` wildcard is definitely supported. Look, but we want to include only modules that can be executed as tests, esp. not the individual group parents, like `integration-test-groups/xml/pom.xml` or `integration-test-groups/http/pom.xml`. Hence I think what could work is the following: 

```json
            "includes": [
                "integration-tests/*/pom.xml",
                "integration-test-groups/*/**/pom.xml",
                "integration-tests-jvm/*/pom.xml"
            ],
```

Reply:
OK, but `"integration-test-groups/*/**/pom.xml"` still includes `integration-test-groups/xml/pom.xml`. So it has to add `"integration-test-groups/*/pom.xml"` in the `excludes`.

Reply:
> OK, but `"integration-test-groups/*/**/pom.xml"` still includes `integration-test-groups/xml/pom.xml`. So it has to add `"integration-test-groups/*/pom.xml"` in the `excludes`.

Sorry about that. With the exclude, we are on the same amount of lines. Feel free to decide which you find more elegant.